### ISSUE TITLE
Regex pattern check causing login error

### ISSUE THREAD

**Issue by Eperty123** [OpenFusionProject/OpenFusion#85]
The new regex patterns:
```
std::regex loginRegex("[a-zA-Z0-9_-]{4,32}");
std::regex passwordRegex("[a-zA-Z0-9!@#$%^&*()_+]{8,32}");
```
give off "Login Error" from the client when logging in. I'm not the biggest regex expert but there seems to be a strict rule for some letter combinations. Using the older patterns fixed the issue however:

```
std::regex loginRegex("^([A-Za-z\\d_\\-]){5,32}$");
std::regex passwordRegex("^([A-Za-z\\d_\\-@$!%*#?&,.+:;<=>]){8,32}$");
```

I believe this is what some of the affected players have been experiencing.

**Comment by gsemaj** [comment]
Have you tried this on the latest commit? There was an issue with bad cookie data causing a lot of login errors that should have been fixed in fe370df. This could very well be a factor though.

**Comment by Eperty123** [comment]
> Have you tried this on the latest commit? There was an issue with bad cookie data causing a lot of login errors that should have been fixed in [fe370df](https://github.com/OpenFusionProject/OpenFusion/commit/fe370df53429d9cc7da4d7cc9a6dbcda273800e8). This could very well be a factor though.

I sure did. Made sure I fetched all the latest commits.

**Comment by CakeLancelot** [comment]
Should be fixed? Haven't had a login error for a long time.
